### PR TITLE
 EZP-26775: supported Solr backend v6

### PR DESCRIPTION
**Pull request by pspanja**
This PR resolves https://jira.ez.no/browse/EZP-26775

`ezplatform-solr-search-engine` counterpart: https://github.com/ezsystems/ezplatform-solr-search-engine/pull/83

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,7 +47,9 @@ matrix:
     - php: 7.0
       env: ELASTICSEARCH_VERSION="1.4.2" TEST_CONFIG="phpunit-integration-legacy-elasticsearch.xml" SYMFONY_VERSION="~2.8.0"
     - php: 7.0
-      env: SOLR_VERSION="4.10.4" TEST_CONFIG="phpunit-integration-legacy-solr.xml" CUSTOM_CACHE_POOL="singleredis" CORES_SETUP="single" SOLR_CONFS="vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/schema.xml vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/custom-fields-types.xml vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/language-fieldtypes.xml"
+      env: SOLR_VERSION="4.10.4" TEST_CONFIG="phpunit-integration-legacy-solr.xml" CUSTOM_CACHE_POOL="singleredis" CORES_SETUP="single" SOLR_CORES="collection1" SOLR_CONFIG="vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/schema.xml vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/custom-fields-types.xml vendor/ezsystems/ezplatform-solr-search-engine/lib/Resources/config/solr/language-fieldtypes.xml"
```
Comment (andrerom) [inline .travis.yml]: We can drop testing of 4.10

File: bin/.travis/prepare_unittest.sh [hunk 2]
```diff
@@ -40,7 +40,7 @@ COMPOSER_UPDATE=""
 # solr package search API integration tests
 if [ "$TEST_CONFIG" = "phpunit-integration-legacy-solr.xml" ] ; then
     echo "> Require ezsystems/ezplatform-solr-search-engine:^1.1.0@dev"
-    composer require --no-update ezsystems/ezplatform-solr-search-engine:^1.1.0@dev
+    composer require --no-update ezsystems/ezplatform-solr-search-engine dev-EZP-26775-solr-6@dev
```
Comment (andrerom) [inline bin/.travis/prepare_unittest.sh]: tmp

**Comment by andrerom** [comment]
can't fully make sense of the failures

**Comment by pspanja** [comment]
It was failing because on the Solr side I didn't base it on the current master, was missing the negative query fix. Now rebased and it passes, though we have OOM with Composer on PHP 5.6, not related to this.

File: .travis.yml (hunk 1 above)
Comment (pspanja) [inline .travis.yml]: Removed Solr `4.10.4` build in https://github.com/ezsystems/ezpublish-kernel/pull/1864/commits/e8d229b4f381d56522eb96920ca128d88cba7596

**Comment by pspanja** [comment]
Added test for Location full text search scoring in https://github.com/ezsystems/ezpublish-kernel/pull/1864/commits/312383aea47dc131dd6b1fa4a81926fa98129f08

**Pull request merged**